test/api.test.ts

```diff
@@ -10,7 +10,7 @@ import { AzureExtensionApiProvider } from "vscode-azureextensionui/api";
 import { nonNullValue, ProjectLanguage, registerOnActionStartHandler } from '../extension.bundle';
```

Comment:
You can ignore the test failures to net5.0 isolated. Those are unrelated to your changes